tests/fixtures/autouse_fixtures.py

```diff
@@ -13,6 +13,11 @@
 from composer.utils import dist, reproducibility
 
 
+@pytest.fixture(autouse=True)
+def disable_tokenizer_parallelism():
+    os.environ['TOKENIZERS_PARALLELISM'] = 'false'
```

Comment:
What does this do? can you maybe leave a comment?

Reply:
added a comment. basically it just silences a warning from huggingface